splash-screen/README.md

```diff
@@ -155,6 +156,34 @@ To use splash screen images named something other than `splash.png`, set `androi
 </style>
 ```
 
+To enable the new recommended **[Android 12 Splash Screen API](https://developer.android.com/guide/topics/ui/splash-screen)** in SDK Versions 30 or below:
+_This is enabled by default and required for SDK version 31+_
+
+- Add `$coreSplashScreenVersion` to the `variables.gradle` file.
+  _See Variables Section for more information_
+
+- Add `implementation "androidx.core:core-splashscreen:$coreSplashScreenVersion"` in the dependencies section of `/android/app/build.gradle`.
+
+- In `android/app/src/main/res/values/styles.xml`, edit the theme `parent` attribute on the Applications's MainActivity Theme to `Theme.SplashScreen` and add desired options to the theme.
+
+```xml
+<style name="AppTheme.NoActionBarLaunch" parent="Theme.SplashScreen">
+    <item name="windowSplashScreenAnimatedIcon">@drawable/splash</item>
+    <item name="windowSplashScreenBackground">@android:color/white</item>
+<!--  !!! OPTIONAL BRANDING IMAGE !!! -->
+<!--  <item name="windowSplashScreenBrandingImage">@drawable/NAME</item> -->
```

Comment:
```suggestion
    <item name="android:background">@drawable/splash</item>
```
I've been testing multiple things and for me the best experience (to be closer to the old behaviour) is to keep the `android:background` and not use the new properties.
Using this, on Android < 12 the behaviour is almost the same as it was before and using the programmatic show will show a very close splash as the initial one.
`android:background` won't work on Android 12+, but by default it will show the round icon there and then the programmatic show will show the old splash. We should document that.